ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessRewriter.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.restmodel.displayedgraph
```

Comment:
I think that all three classes could be in in separate `rewrite` package witch one public class and method. WDYT?